app/scraper.py

```diff
@@ -2,13 +2,15 @@
 import os, json, sys
 import requests
 from bs4 import BeautifulSoup
+from imp import reload
```

Comment:
Just check out whether this is compatible with `python2.7`, because as of `Python3`, the _reload_ function has been moved to `importlib`

Reply:
@Pk13055 yes it's compatible with python 2.7

Reply:
Can you update this to something like  a `try .. except ` block? That way we can maintain cross compatibility. Something like 
```python
try:
    from imp import reload
except ImportError:
    from importlib import reload
```

Reply:
And I should remove sys.setdefaultencoding for python 3 compatibility.